packages/react-dom/src/events/SyntheticKeyboardEvent.js

```diff
@@ -16,6 +16,12 @@ import getEventModifierState from './getEventModifierState';
  */
 const SyntheticKeyboardEvent = SyntheticUIEvent.extend({
   key: getEventKey,
+  code: function(event) {
+    if (event.type === 'keydown' || event.type === 'keyup') {
```

Comment:
Can we just have `code: null`? Why does it need special handling?

Reply:
I assumed this structure have something with react internals or `KeyboardEvent` specification because lower on this interface similar blocks are placed. When `keypress` is executed without react, `code` is equal to output of `keydown` and `keyup` events. However, if it executed with react, the output will be `null`.